### PR TITLE
Added all the basic stuff

### PR DESCRIPTION
**Pull request by GaoYeGithub**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/toronto**](https://counterspell-6olfdup1s.hackclub.dev/toronto) |
|:-:|
|<a href="https://counterspell-6olfdup1s.hackclub.dev/toronto">Screenshot of /toronto</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/wu-z6KhM5xhbUsdtpQ3ffSlqcaBWtgsWmpNroAnAKia878sHvbMWxl3dpOEyZoHsy3s5GqECjw7ihlrss3N1jg.png">(view full size)</a> |







Commit 0bccf7749fd20b531d8d9398d1aaa85b6b061b8f (https://counterspell-6olfdup1s.hackclub.dev).

File: package.json
Comment (samdev-7) [inline package.json:1]: Please revert the change to this file.

File: package-lock.json
Comment (samdev-7) [inline package-lock.json:1]: Please revert the change to this file.

File: yarn.lock
Comment (samdev-7) [inline yarn.lock:1]: Sorry for being picky, please revert the change to this file.

File: .yarnrc.yml
Comment (samdev-7) [inline .yarnrc.yml:1]: Please revert the change to this file, only change what's needed.

File: .gitattributes
Comment (samdev-7) [inline .gitattributes:1]: Please revert the change to this file.

File: components/Schedule.js
Comment (samdev-7) [inline components/Schedule.js:1]: I don't think this was intentional, you should be editing components/Toronto/Schedule.js instead.

Please revert this change.

File: components/Toronto/Locations.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          </p>
+        </div>
+      </div>
+
+      <div className="flex flex-col items-center space-y-3 uppercase">
+        <p className="text-xl text-center">
+          Get notified when registrations open
+        </p>
+        {/* TODO: Replace `Example City` with your city */}
+        <Subscribe eventName="Example City" />
```
Comment (samdev-7) [inline components/Toronto/Locations.js:60]: Please replace this with Toronto

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Some more changes and it's good to go. Let me know if you need anything!

**Comment by GaoYeGithub** [comment]
I think I reverted all the things, please tell me if I missed any!

File: components/Schedule.js
Comment (samdev-7) [inline components/Schedule.js:1]: Seems like this file was still changed for some reason.

File: yarn.lock
Comment (samdev-7) [inline yarn.lock:1]: This file still needs to be reverted.

**Review by samdev-7 — CHANGES_REQUESTED** [review]
A few more changes need to be reverted, sorry!

File: yarn.lock
Comment (GaoYeGithub) [reply yarn.lock:1]: Hmm that weird it looks the same to me, lemme me redo the two files
image

Comment (GaoYeGithub) [reply yarn.lock:1]: I downloaded and move it into my own file but nothing changed
image

**Comment by GaoYeGithub** [comment]
Sorry it should be right now, can you double check?

File: package.json
Comment (samdev-7) [inline package.json:1]: This has been changed now and needs reverting.

Comment (samdev-7) [inline package.json:1]: This has been changed now and needs reverting.

File: components/Schedule.js [hunk 2]
```diff
@@ -30,4 +30,4 @@ export default function Schedule() {
       </div>
     </>
   );
-}
+}
```
Comment (samdev-7) [inline components/Schedule.js:33]: The empty line at the end of the file was removed and needs to be added back.

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Almost! Let me know if you need any help.

p.s. If you're using VS code, you can commit only the files you want by manually clicking the plus sign next to the name in the version control tab!

File: yarn.lock
Comment (samdev-7) [reply yarn.lock:1]: It just needs a blank line added to the end of the file.

**Comment by GaoYeGithub** [comment]
Is it correct now?

**Review by samdev-7 — APPROVED** [review]

**Comment by samdev-7** [comment]
Yup, thanks!

**Pull request merged**